src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -12,6 +12,12 @@
   // Toolbar Item
   --pf-l-toolbar__item--MarginRight: var(--pf-global--spacer--md);
   --pf-l-toolbar__item--MarginLeft: var(--pf-global--spacer--md);
+
+  // immediate descendant selector for toolbar should provide margin by default
+  .pf-l-toolbar__group,
+  .pf-l-toolbar__item {
+    margin-top: 10px;
```

Comment:
In most cases you shouldn't use pixel values as a value for a CSS property. values for CSS properties that a user might want to customize, like spacing, color, etc, should be a CSS variable.

Reply:
Ya you should set the variable value above like the others and use a spacer to do so. I'd suggest using the `--pf-global--spacer--md` to be consistent.

Reply:
Thanks @michael-coker and @matthewcarleton for looking into this. I readjusting the padding and margins again so that they functions as requested. Please review again at your convenience.